### PR TITLE
Remove and replace iota-sdk in the-merge

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of change

Removed and replaced the iota-sdk. Replaced the imports with [@saucelabs/docusaurus-theme-github-codeblock](https://github.com/saucelabs/docusaurus-theme-github-codeblock)

## Type of change

- Enhancement (a non-breaking change which adds functionality)
- Documentation Enhancement

## Change checklist

- [x] I have followed the [contribution guidelines](https://github.com/iota-wiki/iota-wiki/blob/main/.github/CONTRIBUTING.md) for this project
- [x] I have performed a self-review of my own changes
- [x] I have made sure that added/changed links still work
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

**Review by lucas-tortora — CHANGES_REQUESTED** [review]
@Dr-Electron , is there a commit missing? You added the cli-wallet docs, but not the SDK, so we can't really test the `[@saucelabs/docusaurus-theme-github-codeblock](https://github.com/saucelabs/docusaurus-theme-github-codeblock)`

File: next/docusaurus.config.js [hunk 1]
```diff
@@ -265,5 +265,39 @@ module.exports = {
...
+          path: path.resolve(__dirname, 'external/cli-wallet/docs'),
+          routeBasePath: 'cli-wallet',
+          sidebarPath: require.resolve('./external/cli-wallet/sidebars.js'),
+          editUrl: 'https://github.com/iota-wiki/iota-wiki/edit/main/',
+      }
+    ],
+  ],
+  staticDirectories: [
+    path.resolve(__dirname, 'external/iota-sdk/static'),
+    path.resolve(__dirname, 'external/cli-wallet/static'),
```
Comment (lucas-tortora) [inline next/docusaurus.config.js:301]:
```suggestion
  staticDirectories: [
    path.resolve(__dirname, 'external/*/static')
```

**Review by lucas-tortora — CHANGES_REQUESTED** [review]

**Comment by Dr-Electron** [comment]
> @Dr-Electron , is there a commit missing? You added the cli-wallet docs, but not the SDK, so we can't really test the `@saucelabs/docusaurus-theme-github-codeblock`

Wdym? Everything is there? Am I missing something?

**Comment by Dr-Electron** [comment]
Hmmm. I see now. So the docs are there, but somehow not shown correctly? Will check later today...
But if you look at the changes, I can see the docs there... 🤔

**Comment by lucas-tortora** [comment]
>  So the docs are there, but somehow not shown correctly? 

I think it was just a display bug or something, I can see them now  :)

File: next/external/iota-sdk/docs/how_tos/accounts_and_addresses/list_accounts.mdx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/accounts_and_addresses/list_accounts.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/list_accounts.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/accounts_and_addresses/list_accounts.mdx:35]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/list-accounts.ts
```

File: next/external/iota-sdk/docs/how_tos/accounts_and_addresses/list_addresses.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/accounts_and_addresses/list_addresses.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/list_addresses.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/accounts_and_addresses/list_addresses.mdx:33]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/list-addresses.ts
```

File: next/external/iota-sdk/docs/how_tos/accounts_and_addresses/list_outputs.mdx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/accounts_and_addresses/list_outputs.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/list_outputs.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/accounts_and_addresses/list_outputs.mdx:32]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/list-outputs.ts
```

File: next/external/iota-sdk/docs/how_tos/accounts_and_addresses/create_address.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/accounts_and_addresses/create_address.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/create_address.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/accounts_and_addresses/create_address.mdx:46]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/create-address.ts
```

File: next/external/iota-sdk/docs/how_tos/accounts_and_addresses/check_balance.mdx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/blob/develop/sdk/examples/how_tos/accounts_and_addresses/check_balance.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/check_balance.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/accounts_and_addresses/check_balance.mdx:50]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/accounts_and_addresses/check-balance.ts
```

File: next/external/iota-sdk/docs/how_tos/outputs/unlock_conditions.mdx [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/outputs/unlock_conditions.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/outputs/unlock_conditions.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/outputs/unlock_conditions.mdx:50]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/outputs/unlock-conditions.ts
```

File: next/external/iota-sdk/docs/how_tos/sign_and_verify_ed25519/sign_ed25519.mdx [hunk 8]
```diff
@@ -0,0 +1,55 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/sign_and_verify_ed25519/sign_ed25519.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/sign_ed25519.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/sign_and_verify_ed25519/sign_ed25519.mdx:36]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/sign-ed25519.ts
```

File: next/external/iota-sdk/docs/how_tos/sign_and_verify_ed25519/verify_ed25519_signature.mdx [hunk 9]
```diff
@@ -0,0 +1,53 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/sign_and_verify_ed25519/verify_ed25519_signature.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/verify_ed25519_signature.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/sign_and_verify_ed25519/verify_ed25519_signature.mdx:36]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/sign_and_verify_ed25519/verify-ed25519-signature.ts
```

File: next/external/iota-sdk/docs/how_tos/simple_transaction/request_funds.mdx [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/simple_transaction/request_funds.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/simple_transaction/request_funds.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/simple_transaction/request_funds.mdx:46]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/simple_transaction/request-funds.ts
```

File: next/external/iota-sdk/docs/how_tos/simple_transaction/simple_transaction.mdx [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+
+```rust reference
+https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples/how_tos/simple_transaction/simple_transaction.rs
+```
+
+</TabItem>
+<TabItem value="nodejs" label="Nodejs">
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/simple_transaction/simple_transaction.ts
```
Comment (lucas-tortora) [inline next/external/iota-sdk/docs/how_tos/simple_transaction/simple_transaction.mdx:46]:
```suggestion
https://github.com/iotaledger/iota-sdk/tree/develop/bindings/nodejs/examples/how_tos/simple_transaction/simple-transaction.ts
```

File: next/external/iota-sdk/static/img/iota_layers_overview.svg
Comment (lucas-tortora) [inline next/external/iota-sdk/static/img/iota_layers_overview.svg:1]: didn't this get updated ?

Comment (Dr-Electron) [reply next/external/iota-sdk/static/img/iota_layers_overview.svg:1]: I'm never sure about it 😆 But seems not. I also asked Thibault if we should update to the SDK layout in general

**Review by jlvandenhout — APPROVED** [review]
Other then the issues with the SDK content itself that we should probably fix upstream for now, like `/iota-sdk/how_tos/accounts_and_addresses/check_balance` crashing and some examples with a wrong URL, having the content in the Wiki repo and importing remote code seem to work quite well 👍

**Review by lucas-tortora — APPROVED** [review]
we should look into why how_tos/accounts_and_addresses/check_balance/ crashes due to CORS, but all in all, I think we can merge this into `the-merge` and sort it out in another PR, wdyt?

**Comment by jlvandenhout** [comment]
> we should look into why how_tos/accounts_and_addresses/check_balance/ crashes due to CORS, but all in all, I think we can merge this into `the-merge` and sort it out in another PR, wdyt?

Not sure if those are related, though. I get CORS errors all over the place, and that's a server thing, not a client thing. `how_tos/accounts_and_addresses/check_balance` doesn't really crash for me. It just doesn't show any content.

But I agree that is unrelated to this PR, we can merge and then investigate separately.

**Pull request merged**